plugins_src/import_export/wpc_lwo.erl

```diff
@@ -52,32 +61,57 @@ props(import) ->
 props(export) ->
     [{ext, ".lwo"},{ext_desc, ?__(1,"LightWave File")}].
 
+-ifdef(DEBUG).
 
-%%% ===============================
-%%% === LightWave Export (LWO2) ===
-%%% ===============================
+-define(DEBUG_TAG(Item), begin
+    io:fwrite("       ~p\n", [Item]),
+    Item
+end).
 
-export(E3DFile) ->
-    LWO = make_lwo(E3DFile),
-    file:write_file("1.lwo", LWO).
+-define(DEBUG(Item), begin
+    io:fwrite("~s\n", [Item]),
+    ok
+end).
 
-export(FileName, Contents) ->
-    LWO = make_lwo(Contents),
-    file:write_file(FileName, LWO).
+-define(DEBUG_B(Item), begin
+    if
+        is_atom(Item) -> String = atom_to_list(Item);
+        is_list(Item) -> String = Item
+    end,
+    NumChars = length(String),
+    io:fwrite("+-~s-+\n", [lists:duplicate(NumChars, "-")]),
+    io:fwrite("| ~s |\n", [String]),
+    io:fwrite("+-~s-+\n", [lists:duplicate(NumChars, "-")]),
+    ok
+end).
 
-make_lwo(Contents0) ->
-    FlipX = e3d_mat:scale(-1.0, 1.0, 1.0),
-    Contents = e3d_file:transform(Contents0, FlipX),
-    #e3d_file{objs=Objs,mat=Mats,creator=Creator} = Contents,
-    Tags = make_tags(Mats),
-    Surfs = make_surfaces(Mats),
-    Maker = make_auth(Creator),
-    MaterialDict = make_material_dict(Mats),
-    ObjectChunks = make_objects_chunk(Objs, MaterialDict),
-    Chunks = [Maker, Tags, ObjectChunks|Surfs],
-    LwoHeader = make_header(Chunks),
-    LwoData = [LwoHeader|Chunks],
-    LwoData.
+-else.
+
+-define(DEBUG_TAG(Item), Item).
+
+-define(DEBUG(Item), ok).
+
+-define(DEBUG_B(Item), ok).
+
+-endif.
+
+
+-define(SINT, big-signed-integer).
+-define(UINT, big-unsigned-integer).
+-define(UW, :16/?UINT).
+-define(FLT, :32/big-float).
+-define(FLT8, :64/big-float).
+-define(ID, :4/bytes).
```

Comment:
Nitpick, I would prefer if you removed the `:` from the macros, I don't like `Var?Macro` constructs, I think `Var:?Macro` is better.
So the macro is its own entity.